### DISCUSSION TITLE
Drag and Drop to Folder

### DISCUSSION THREAD

**Discussion by Airr** [massCodeIO/massCode#473] [Ideas]
I like the way you can drag and drop code or a file into the snippet area, but is there a possibility to drag and drop a file to a folder and have an entry created using the filename with the snippet populated?

Great product, btw.  I use it on my macOS and Windows systems.

**Comment by pa-0** [comment]
That's pretty cool. I did not realize you could do that until reading this!

Does massCode support dragging/dropping different filetypes into markdown documents like some editors do?  Any file dropped into the document is copied to an `.attachments` folder in the same directory where the related markdown file is stored and the editors can simultaneously generate a relative link to the file in the markdown like so:
```markdown
[filename](.\link\to\file.zip)
```
if the file is binary, e.g., executable, zip, etc.  Or if the file type is (dis)playable media, the link is formatted as below
```markdown
![media](.\link\to\image-video-or-audio-file.mp4)
```
so that the image can be rendered in the preview pane of the editor (and/or played directly therefrom if the file contains video/audio). Pretty nifty feature, especially where there is a GitHub support since the files are then automatically added to the same repo where the markdown files are hosted.
